### PR TITLE
Support redux-persist v4

### PR DESCRIPTION
**Pull request by richardscarrott**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,19 +28,19 @@
...
   },
   "devDependencies": {
     "babel-cli": "^6.11.4",
     "babel-preset-es2015": "^6.13.2",
     "babel-register": "^6.11.6",
     "chai": "^3.5.0",
     "mocha": "^3.0.2",
-    "release-it": "^2.4.3"
+    "release-it": "^2.4.3",
+    "redux-persist": "^4.0.1"
```
Comment (richardscarrott) [inline package.json]: I moved redux-persist to dev deps to allow the consumer app to use v3 or v4 without pulling in both versions.

**Comment by edy** [comment]
thank you. merged.

**Comment by richardscarrott** [comment]
Thanks @edy, would you be able to publish this when you get some time?

**Comment by edy** [comment]
done

**Pull request merged**